docs/devguide/introduction.rst

```diff
@@ -4,34 +4,88 @@
 Introduction
 ############
 
+.. _introduction--what-is-opendds:
+
 ****************
 What is OpenDDS?
 ****************
 
 ..
     Sect<0.1>
+    Sect<0.6>
+
+OpenDDS is an open-source C++ framework for exchanging data in distributed systems.
+It is an implementation of :ref:`a group of related OMG specifications <specs>`.
+OpenDDS is implemented in C++ and contains support for :doc:`Java <java_bindings>`.
+Users in the OpenDDS community have contributed and maintain bindings for other languages that include `C# <https://www.openddsharp.com/>`__, `Node.js <https://github.com/OpenDDS/node-opendds>`__, and `Python <https://github.com/OpenDDS/pyopendds>`__.
+OpenDDS is sponsored by the OpenDDS Foundation and is available via https://opendds.org and https://github.com/OpenDDS/OpenDDS.
 
-OpenDDS is an open source implementation of a group of related Object Management Group (OMG) specifications.
+There is an FAQ available at https://opendds.org/faq.html.
```

Comment:
Do you think we should eventually move the FAQ to the dev guide?

Reply:
Probably, but there's already quite a bit in this PR.

Reply:
Absolutely.  That's for another day.